spec/rubocop/cop/rails/delegate_spec.rb

```diff
@@ -42,6 +42,18 @@ def foo
     RUBY
   end
 
+  it 'does not find trivial delegate to `self` when `module_function` is used' do
+    expect_no_offenses(<<~RUBY)
+      module A
+        module_function
+
+        def foo
+          self.foo
+        end
+      end
+    RUBY
+  end
```

Comment:
Can you also add a spec like the following as an example?

```ruby
module A
  module_function def foo
    self.foo
  end
end
```

Reply:
Added.